internal/connector/pingfederate/resources/pingfederate_authentication_api_settings.go

```diff
@@ -0,0 +1,53 @@
+package resources
+
+import (
+	"github.com/pingidentity/pingctl/internal/connector"
+	"github.com/pingidentity/pingctl/internal/connector/common"
+	"github.com/pingidentity/pingctl/internal/logger"
+)
+
+// Verify that the resource satisfies the exportable resource interface
+var (
+	_ connector.ExportableResource = &PingFederateAuthenticationApiSettingsResource{}
+)
+
+type PingFederateAuthenticationApiSettingsResource struct {
+	clientInfo *connector.PingFederateClientInfo
+}
+
+// Utility method for creating a PingFederateAuthenticationApiSettingsResource
+func AuthenticationApiSettings(clientInfo *connector.PingFederateClientInfo) *PingFederateAuthenticationApiSettingsResource {
+	return &PingFederateAuthenticationApiSettingsResource{
+		clientInfo: clientInfo,
+	}
+}
+
+func (r *PingFederateAuthenticationApiSettingsResource) ExportAll() (*[]connector.ImportBlock, error) {
+	l := logger.Get()
+
+	importBlocks := []connector.ImportBlock{}
+
+	l.Debug().Msgf("Generating Import Blocks for all %s resources...", r.ResourceType())
+
+	authnApiSettingsId := "authentication_api_settings_singleton_id"
+	authnApiSettingsName := "Authentication API Settings"
+
+	commentData := map[string]string{
+		"Resource Type": r.ResourceType(),
+		"Authentication API Settings Resource ID":   authnApiSettingsId,
```

Comment:
Maybe we just leave this off? This value is meaningless. Or maybe replace this with a comment just saying this is a singleton resource?

Reply:
Updated!